docs/release-notes/keyboard_from_live.rst

```diff
@@ -0,0 +1,22 @@
+:Type: Live Environment
+:Summary: Use keyboard layout configuration from the Live system
+
+:Description:
+    Until now Anaconda, it was required to specify keyboard layout for the Live environment.
+    With this change, live system itself is responsible for the keyboard configuration and
+    Anaconda just reading the configuration from the live system for the installed system.
+
+    This works only if it is not specified by user and only Gnome Shell environment is
+    supported so far. Spins should be added later.
+
+    This is proper fix for https://bugzilla.redhat.com/show_bug.cgi?id=2016613 where we found
+    that Wayland don't allow Anaconda to control keyboard configuration. The original fix
+    separated installation and current environment which is wrong because user can type
+    passwords with other keyboard layout than what will be installed. This solution will leave
+    the control on the system itself and just reads the configuration.
+    It is also a big step to resolve https://bugzilla.redhat.com/show_bug.cgi?id=1955025 .
```

Comment:
I'm not sure if we want to mention all of that? If so, it needs more proofreading which I can't do right now.

Reply:
Good point. My reasoning for this was to have information why we need to do this change. Also explain why these bugs are attached because they are not directly solving this.